.travis.yml

```diff
@@ -0,0 +1,10 @@
+language: node_js
+sudo: false
+node_js:
+  - "iojs"
+  - "0.12"
+  - "0.10"
+script:
+  - npm run peasant -- lint test build
+after_script:
+  - npm run coveralls
```

Comment:
Can you turn off email notifications? Also please add slack notifications to blueprint-notify channel.